lib/git-fastclone.rb

```diff
@@ -337,21 +336,13 @@ def prefetch(submodule_file)
     # that this repo has been updated on this run of fastclone
     def store_updated_repo(url, mirror, repo_name, fail_hard)
       unless Dir.exist?(mirror)
-        fail_pipe_on_error(
-          ['git', 'clone', verbose ? '--verbose' : '--quiet', '--mirror', url.to_s,
-           mirror.to_s], quiet: !verbose
-        )
+        fail_on_error('git', 'clone', verbose ? '--verbose' : '--quiet', '--mirror', url.to_s, mirror.to_s,
+                      quiet: !verbose)
       end
 
       Dir.chdir(mirror) do
         cmd = ['git', 'remote', verbose ? '--verbose' : nil, 'update', '--prune'].compact
-        if verbose
```

Comment:
@gyfelton Why do we use `!verbose` for both conditions? Should it be `verbose`?

Reply:
see the comment in else block. i think it's just a matter of testing out what happens when the master is very old and one perform `git remote --verbose update --prune`
happy to pair on this to re-trigger my memory

Reply:
Ah, I see now.